backend/btrixcloud/db.py

```diff
@@ -32,7 +32,7 @@
     ) = PageOps = BackgroundJobOps = object
 
 
-CURR_DB_VERSION = "0043"
+CURR_DB_VERSION = "0044"
```

Comment:
Looks like maybe the migration that goes with this version bump hasn't been committed?